tokio/Cargo.toml

```diff
@@ -86,6 +87,7 @@ futures-util-preview = { version = "=0.3.0-alpha.19", features = ["sink"] }
 
 # Everything else is optional...
 bytes = { version = "0.4", optional = true }
+crossbeam-channel = { version = "0.3.8", optional = true }
```

Comment:
Not sure I follow why this was added?

Reply:
current-thread needs it IIRC.

Reply:
Oh, I guess this was in `tokio-executor`'s `Cargo.toml` before then?

Reply:
yes